controllers/remove_process_groups.go

```diff
@@ -293,59 +301,63 @@ func includeProcessGroup(ctx context.Context, logger logr.Logger, r *FoundationD
 		return err
 	}
 
-	// Reset the SecondsSinceLastRecovered sine the operator just included some processes, which will cause a recovery.
+	// Reset the SecondsSinceLastRecovered since the operator just included some processes, which will cause a recovery.
 	status.Cluster.RecoveryState.SecondsSinceLastRecovered = 0.0
+	// Update the process group list and remove all removed and included process groups.
+	cluster.Status.ProcessGroups = newProcessGroups
```

Comment:
In a follow up PR I'll create some additional checks in our e2e tests to make sure all processes are included later, but that will required some fine tuning to make sure we are not hitting timeouts with the tests.

Reply:
Created: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/2149